tests/test_grammar.py

```diff
@@ -199,9 +199,109 @@ def test_normalize(type_str, normalized):
     assert normalize(type_str) == normalized
 
 
+@pytest.mark.parametrize(
+    'type_str',
+    (
+        'bytes[2][2]',
+        'bytes',
+        'string',
+        '(int[],bool)',
+    ),
+)
+def test_abi_type_item_type(type_str):
+    abi_type = parse(type_str)
+    assert not abi_type.has_dynamic_arrlist
```

Comment:
This test looks repeated. Based on the name, maybe it was supposed to test something else?

Reply:
Whoops!